test/usage-test.js

```diff
@@ -2,42 +2,42 @@
 const tape = require('tape');
 const common = require('./common');
 
-function removeBlankLines(lines) {
-  return lines.filter((line) => { return line.trim() !== ''; });
-}
-
 tape('usage messages', (t) => {
   t.timeoutAfter(15000);
 
   const sess = common.Session.create('inspect-scenario.js');
 
-  sess.waitBreak(() => {
+  sess.waitBreak((err) => {
+    t.error(err);
     sess.send('v8 print');
   });
 
-  sess.stderr.linesUntil(/USAGE/, (lines) => {
-    t.ok(/^error: USAGE: v8 print expr$/.test(removeBlankLines(lines)[0]),
-         'print usage message');
+  sess.stderr.linesUntil(/USAGE/, (err, lines) => {
+    t.error(err);
+    const re = /^error: USAGE: v8 print expr$/;
+    t.ok(lines.some(line => re.test(line.trim())), 'print usage message');
```

Comment:
Since `lines.some(line => re.test(line.trim()))` is repeated a number of times, it might be better to keep it as a helper function that you can pass `re` and `lines` to.